doc/file_format.rst

```diff
@@ -0,0 +1,17 @@
+File format reference
```

Comment:
Data set format?

The system engineers might call this our Science Data Model (I got asked about this just two days ago 😄).

Reply:
Yep, changed to data set format.